docs/topics/asset/asset.md

```diff
@@ -0,0 +1,55 @@
+---
+description: Asset
+---
+
+# Asset
+
+## Introduction
+
```

Comment:
Add one sentence on what is an Asset (product wise) & links to the user manual
https://sandboxgame.gitbook.io/the-sandbox/assets/what-are-assets